tasks/adventure_solutions/task_5_loop_range_function.py

```diff
@@ -0,0 +1,69 @@
+import random
+import time
+
+def wrong_input(valid):
+    print(f"Sorry but the only valid options are: {valid}.")
+
+# create `dead` function with `death_message` argument
+def dead(death_message):
+    print(f"Sorry {name}, {death_message}.")
+    print("You're falling for...")
+
+    # print numbers slowly
+    for number in range(1,11):
+        print(number)
+        time.sleep(1)
```

Comment:
```suggestion
        time.sleep(0.3)
```
This way it is a bit faster in the game play and it could print even more numbers